release/tag/goreleaser.yaml

```diff
@@ -19,15 +19,26 @@ builds:
     goos:
       - darwin
       - linux
-      - windows
```

Comment:
can you update installation doc as part of this PR:

https://kpt.dev/installation/

I agree with not having windows builds until it's tested and works.

Reply:
Providing links that always point to the latest release is actually a bit tricky. I want to only publish binaries through github, but since we make releases from two brances, the `latest` release doesn't always point to what we want. The releases for the example packages also creates some issues here, also with regards to the change list.

Reply:
I wouldn't tell user to go the release tab in GitHub UI.

The docs will also be versioned (Imagine a drop down in kpt.dev with a list of releases). In that case, the links in the installation doc should point to a specific release right?

We can take this outside the scope of this PR but need to figure it out.

@etefera

Reply:
We will have links directly to the binaries in the documentation. We'll just have those hosted as part of the github releases instead of in GCS. We can provide direct links for download, which would fix the issue of using the generic link to point to the latest github release artifacts. For bugfix releases that probably will not have a separate version of the documentation, we can automate updating the links.

Reply:
Please follow up and make sure we have tracking issue for this work.

Reply:
Updated the installation page to reflect how users can get kpt from the next branch now. This means commenting out information about gcloud and homebrew.
The links must be updated once we have a kpt release from the `next` branch.